src/components/CommandBar.js

```diff
@@ -53,10 +97,14 @@ const CommandBar = React.createClass({
 
   componentWillUnmount() {
     const shortcuts = this.context.shortcuts;
-    shortcuts.off("F8");
-    shortcuts.off("F10");
-    shortcuts.off(`${ctrlKey}F11`);
-    shortcuts.off(`${ctrlKey}Shift+F11`);
+    ["resume", "stepOver", "stepIn", "stepOut"].forEach(
```

Comment:
lets have fun with this:

in the module context you can set this list

```js
const debuggerCommands = ["resume", "stepOver", "stepIn", "stepOut"];
```

and here it becomes cleaner:

```js
debuggerCommands.forEach(
  action => shortcuts.off(getKeyForOS("WINNT", action))
);
```